cli/utilities/packages.py

```diff
@@ -83,6 +83,30 @@ def install(self, pkg, nogpgcheck=False, env_vars={}):
         elif out:
             raise PackageError(f"Failed to install package '{pkg}'")
 
+    def remove(self, pkg, nogpgcheck=False, env_vars={}):
+        """Remove a package or packages
+
+        Args:
+            pkg (str): package needs to be removed
+            env_vars (dict): dictiory with environment variables
+        """
+
+        cmd = ""
+        if env_vars:
+            for k, v in env_vars.items():
+                cmd += f"{k}={v} "
```

Comment:
Instead use - https://github.com/red-hat-storage/cephci/blob/master/cli/utilities/utils.py#L270

Reply:
It's been resolved now.